server/go.mod

```diff
@@ -53,6 +53,7 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	gotest.tools/v3 v3.1.0
+	sigs.k8s.io/yaml v1.2.0
```

Comment:
You added this dependency, but you are actually using `gopkg.in/yaml.v2` instead. Can this be removed?

Reply:
No, this is used in tests: https://github.com/kubeshop/tracetest/pull/2039/files#diff-4113c542a74918b6256e4f88046a3a8e35994cc244a0747a7c0f7193c8d53961R51